firstpr/vera.md

```diff
@@ -0,0 +1,6 @@
+Hi,I'm Chioma Vera Nkanmuo, Beginner Web developer from Nigeria with basic knowledge of  HTML and  CSS. I am currently in my penultimate level in university.
+Aside  from coding, I love to  read alot.
+FUN FACT: I am a fitness ethusiam.
```

Comment:
Minor: typo `entusiast`. 
No issues, can merge for now.

Reply:
> Minor: typo `enthusiast`.
> No issues, can merge for now.

SarathKumar6 thanks pointing out the typo